src/handler-builder.ts

```diff
@@ -74,8 +83,24 @@ export class HandlerBuilder {
     };
   }
 
+  /**
+   * Create a handler for Realtime Database events.
+   * 
+   * `ref.onCreate` handles new data creation.
+   * `exports.myFunction = functions.handler.database.ref.onCreate((snap, context) => { ... })`
+   
+   * `ref.onUpdate` handles data updates.
+   * `exports.myFunction = functions.handler.database.ref.onUpdate((change, context) => { ... })`
+   
+   * `ref.onDelete` handles data deletion.
+   * `exports.myFunction = functions.handler.database.ref.onDelete((snap, context) => { ... })`
+   
+   * `ref.onWrite` handles data creation, update, or deletion.
+   * `exports.myFunction = functions.handler.database.ref.onWrite((change, context) => { ... })`
+   */
   get database() {
     return {
+      /** @hidden */
```

Comment:
Can you explain why this is hidden, but the return.getDocument for Firestore isn't?  I'm not clear on the pattern.

Reply:
because we don't actually want people to use "functions.handler.database.instance.ref" since "instance" is redundant, it behaves the same as "functions.handler.database.ref". So we don't want to document it, but we do want to still leave the code there so that if someone does write "functions.handler.database.instance.ref" it will still work.

Reply:
Got it, thanks!